dist/stat_data.py

```diff
@@ -727,6 +727,8 @@ def __init__(self, name, desc, flags=''):
     # Compression statistics
     ##########################################
     CompressStat('compress_read', 'compressed pages read'),
+    CompressStat('compress_precomp_intl_max_page_size', 'compressed page pre-compressed internal max size', 'no_clear,no_scale,size'),
+    CompressStat('compress_precomp_leaf_max_page_size', 'compressed page pre-compressed leaf max size', 'no_clear,no_scale,size'),
```

Comment:
I should have made a concrete suggestion for the text, since I'm obviously feeling pedantic:

`compressed page maximum leaf page size prior to compression`

reads better to me.

Reply:
8321182
Done!
I was trying to keep as much of the prefix identical as possible with the previous one.